{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "etr": "2020-12-07T00:00:00-0400",
        "start": "2020-12-06T20:12:00-0400"
      },
      "geom": {
        "p": [
          "_xnkGhc{iK"
        ]
      },
      "id": "2",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
